src/Input/KeyInterop.ts

```diff
@@ -170,8 +170,9 @@ module Fayde.Input {
             }
             return args;
         }
+		
         CreateArgsDown(e): Fayde.Input.KeyEventArgs {
-            if (e["char"] && e.keyCode !== 8 && e.keyCode !== 9)
+            if (e["char"] && e.keyCode !== 8 && e.keyCode !== 9 && !e.ctrlKey)

```

Comment:
This looks fine.  I added some comments to the code to explain that Ctrl+[key] combinations don't flow through press.